docker/s6/services/lazyraster.svc/run

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+# Add the current host as a Memberlist cluster seed when running under Mesos
+MESOS_ML_PORT=$(env | grep "MESOS_PORT_${MEMBERLIST_PORT:-7946}" | cut -d "=" -f 2)
```

Comment:
This works, but it's a bit hackish. I can also switch the script to `/bin/bash` and use a dynamic variable instead. WDYT?